examples/playground/components/website-analyzer-form.tsx

```
@@ -0,0 +1,91 @@
+  isLoggedIn,
+}: {
+  isLoggedIn: boolean;
+}) {
+  const [formError, setFormError] = useState<string | null>(null);
+  const router = useRouter();
+  const supabase = createClient();
+
+  async function handleAnalyzeWebsite(formData: FormData) {
+    const url = formData.get('url') as string;
+    
+    if (!url) {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)